### ISSUE TITLE
Pager - accessibility problem with item per page select 

### ISSUE THREAD

**Issue by michalgarnczarski** [Sage/carbon#5710]
### Current behaviour

A Select with items per page hasn't a label: 

![image](https://user-images.githubusercontent.com/104442068/209841845-9062a6e0-715c-417f-8d12-3dd6fff00cae.png)

### Expected behaviour

The label should be provided out of the box or it should be possible to add it. 

### CodeSandbox or Storybook URL

https://codesandbox.io/s/rough-wildflower-oz70f0?file=/src/index.js

### JIRA Ticket (Sage Only)

SBS-48621, SBS-48585

### Carbon Version

111.8.0

### What browsers are you seeing the problem on?

Firefox, Chrome

### What Operating System are you seeing the problem on?

Windows

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by tomdavies73** [comment]
@michalgarnczarski We believe this issue has been addressed in v111.21.1, please check again using this version. Thanks

**Comment by przemyslawzalewski** [comment]
Regarding the above, I've tried to repro with the latest as of today (v111.22.2) at https://codesandbox.io/s/vigorous-elbakyan-3y4te3 and I can confirm the reported issue is no longer present, thanks!

<img width="1440" alt="Zrzut ekranu 2023-01-4 o 09 07 55" src="https://user-images.githubusercontent.com/30114244/210510727-191672e3-b03a-4d45-aab7-bdb56a14e331.png">

**Comment by tomdavies73** [comment]
Thanks @przemyslawzalewski for the update! I'll close this issue, please feel free to raise another issue if something similar occurs or if any new bugs come to light @michalgarnczarski. Thanks.
